common/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -30,7 +32,7 @@ const ConfirmationDialog = ({
         <Button variant="secondary" onClick={onCancel}>
           {cancel}
         </Button>
-        <Button variant="default" onClick={onConfirm}>
```

Comment:
Also onclick still works even though the button is grayed out. Not sure if that is intended

Reply:
It doesn't work when it's disabled. Have you tested locally and encountered that it still works even though it's disabled?